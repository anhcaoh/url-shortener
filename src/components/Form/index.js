import React from "react";
import "./form.scss";
const Form = ( props ) => {
    const onSubmitHandler = (event) => {
        event.preventDefault();
        return event;;
    };
    return(<>
        <form onSubmit={onSubmitHandler} {...props}>
           {props.children}
        </form>
</>);
};
export default Form;