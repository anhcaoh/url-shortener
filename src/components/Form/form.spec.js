import React from "react";
import renderer from "react-test-renderer";
import Form from "./index";

describe("Form", () => {
  it("should render snapshot", () => {
    const component = renderer.create(<Form>Children within Form</Form>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});