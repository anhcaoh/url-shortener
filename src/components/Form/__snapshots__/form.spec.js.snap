// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  Children within Form
</form>
`;
